components/button.css

```diff
@@ -0,0 +1,96 @@
+.button_main {
+  border: 1px solid var(--gray-border);
+  border-radius: 5px;
+  margin: var(--margin-width);
+  padding: 1rem;
+  display: flex;
+}
+.button {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  padding: 1rem 1.5rem;
+}
+
+.btn-primary {
+  border: 1px solid var(--gray-border);
+  padding: 0.5rem 1rem;
+  color: var(--white);
+  line-height: 1.75;
+  font-weight: 600;
+  letter-spacing: 0.02857em;
+  border-radius: 7px;
+  cursor: pointer;
+}
+.btn-text {
+  background-color: transparent;
+  cursor: pointer;
+  border: 0;
+  line-height: 1.75;
+  font-weight: 600;
+  letter-spacing: 0.02857em;
+}
+.btn-secondary,
+.btn-icon {
+  background-color: transparent;
+  cursor: pointer;
+  border: 0;
+  line-height: 1.75;
+  font-weight: 600;
+  letter-spacing: 0.02857em;
+  border: 1px solid;
+  border-radius: 7px;
+  padding: 0.45rem 1rem;
+}
+.btn-bg-color {
+  background-color: var(--primary-color);
+}
+.action-button {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  padding: 1rem 1.5rem;
+}
+.btn-fab {
+  border: 1px solid var(--primary-color);
+  width: 40px;
+  height: 40px;
+  border-radius: 50%;
+  line-height: 1.5;
+  text-align: center;
+}
+.btn-color {
+  color: var(--primary-color);
+}
+.fab-bg-color {
+  color: var(--primary-color);
+  background-color: transparent;
+}
+
+@media only screen and (max-width: 2000px) and (min-width: 551px) {
+  .button {
+    flex-direction: row;
+    width: 100%;
+    justify-content: space-evenly;
+  }
+  .action-button {
+    flex-direction: row;
+    width: 100%;
+    justify-content: space-evenly;
+  }
```

Comment:
You can avoid code duplication here to reduce your code

Reply:
hey kart-c done with the changes you suggested. could you approve now